### PR TITLE
[Application] LOSS class for YOLO v3 in torch @open sesame 09/14 14:01

### PR DESCRIPTION
**Pull request by baek2sm**
Loss class for YOLO v3. 

Signed-off-by: SeungBaek <baek2sm@gmail.com>

### CODE REVIEW

**Comment by songgot** [comment]
please git rebase yolov3_dknet53_torch https://github.com/nnstreamer/nntrainer/pull/2217 , thank you.

**Review by myungjoo — APPROVED** [review]

File: Applications/YOLOv3/PyTorch/main.py [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+import random
+import sys
+import os
+
+from PIL import Image, ImageDraw
+import matplotlib.pyplot as plt
+
+from yolo import YoloV3
+from yolo_v2 import YoloV2
```
Comment (lhs8928) [inline Applications/YOLOv3/PyTorch/main.py:26]: Do we need YoloV2 to run YoloV3?

File: Applications/YOLOv3/PyTorch/main.py [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+anchors = torch.FloatTensor([
+            [[116, 90], [156, 198], [373, 326]], # large
+            [[30, 61], [62, 45], [59,119]], # medium
+            [[10, 13], [16, 30], [33,23]] # small
+        ]) / 416
+
+epochs = 1000
+batch_size = 8
+
+train_img_dir = '/home/seungbaek/projects/dataset_det/train_dir/images/*'
```
Comment (lhs8928) [inline Applications/YOLOv3/PyTorch/main.py:62]: Please change dir path

**Review by lhs8928 — APPROVED** [review]
LGTM

File: Applications/YOLOv3/PyTorch/main.py (hunk 1 above)
Comment (baek2sm) [reply Applications/YOLOv3/PyTorch/main.py]: We don't need it. Thanks!

File: Applications/YOLOv3/PyTorch/main.py (hunk 2 above)
Comment (baek2sm) [reply Applications/YOLOv3/PyTorch/main.py]: I corrected it. thank you!

**Comment by baek2sm** [comment]
It is already merged by #2229

**Pull request closed without merging**